yarn.lock

```diff
@@ -23377,7 +23377,16 @@ terser-webpack-plugin@^2.1.2, terser-webpack-plugin@^2.3.1:
     terser "^4.4.3"
     webpack-sources "^1.4.3"
 
-terser@^4.1.2, terser@^4.4.3, terser@^4.6.11, terser@^4.6.3:
+terser@4.6.13:
```

Comment:
This should be deduplicated. I can help you with that once you have the other changes in.